### PR TITLE
Tips update

### PR DESCRIPTION
**Pull request by XSlayer300**
## Changelog
:cl:
fix: Tips are updated, removed some outdated tips.
/:cl:

### CODE REVIEW

**Review by TiviPlus — APPROVED** [review]

**Comment by BraveMole** [comment]
Maybe some tips regarding new additions?

Like, turrets in open ground are bad,
you can make small silos
their is a xeno queue
psy points are reseted on hijack
cocoon are opening on hijack

**Comment by Chaznoodles** [comment]
Why the strange differentiation with 'combat marines'? All planetside marines are combatants.

File: strings/tips/meta.txt [hunk 1]
```diff
@@ -13,10 +13,10 @@ While alive, use 'resist' (Default: B) to stop drop and roll when on fire. Human
...
 When in doubt: Adminhelp for problems in common for people violating the rules, Mentorhelp for problems that you want to know.
 Check the SS13 Webmap to learn the layout of the ship or the map currently in play.
-After you ghosted out of the game (either from dying, cryo or manual ghost), you can respawn back into the game after waiting for 30 minutes (as a human) or 5 minutes (as a xenomorph) if there is a larva.
+After you ghosted out of the game (either from dying, cryo or manual ghost), you can respawn back into the game after waiting for 30 minutes (as a human) or 2 minutes (as a xenomorph) if there is a larva.
 To sing, add "%" before any text. Singing on radios will not work, but singing on intercomms will.
 You can respawn during End-Of-Round Deathmatch in the OOC tab.
-Sundering is a concept that reduces the effectiveness of xeno armor by percentage. Xenomorph drones can heal and decrease sunder.
+Sundering is a concept that reduces the effectiveness of xeno armor by percentage. Xenomorph drones can heal and decrease sunder. Resting heal sunder, but slowly.
```
Comment (Chaznoodles) [inline strings/tips/meta.txt:19]: heals

**Review by Chaznoodles — COMMENTED** [review]
Spelling error

**Comment by BraveMole** [comment]
You forgot some too: 
Not being ready at the start of the round means xenos get more burrowed larva, and that you don't get to play as quickly. (whatt?)

As the RO, things get more expensive the more often you buy it. ???

As a ravager, targeting hands or feet is better to inflict a lot of damage while lunging. (lunging ? )

As the Queen, use hive status to quickly swap between alien 'cameras'. removed

As a boiler, your shots are more accurate when fired diagonally. all shot are accurate

As a hivelord, you can upgrade existing weaker resin doors and walls by activating them in build mode via middle/shift click. Can you really? 

As a praetorian, your sticky resin is vastly inferior to that of drones and hivelords. Maybe a joke, dunno

As a runner, plasma gain is halted when you hide. hiding as a runner? 

As a xeno, it's not a good idea to kill hosts during the crash phase if they can be captured instead. this one must be really old

**Comment by XSlayer300** [comment]
> Why the strange differentiation with 'combat marines'? All planetside marines are combatants.

Leftover from the previous tip, I am going to rename them back to squad marine.

**Comment by XSlayer300** [comment]
> As a praetorian, your sticky resin is vastly inferior to that of drones and hivelords. 

Likely referring to sticky spit.

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**